src/components/Breadcrumb/Breadcrumb.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+const Breadcrumb = styled.div`
+  float: left;
```

Comment:
`float` modifies the normal page flow, you should also include a clearfix to reset the flow afterwards

```
&:after {
   clear: both;
}
```